gapis/resolve/state_tree.go

```diff
@@ -242,7 +242,7 @@ func (n *stn) buildChildren(ctx context.Context, tree *stateTree) {
 				children = append(children, &stn{
 					name:           fmt.Sprintf("[%d - %d]", n.subgroupOffset+s, n.subgroupOffset+e-1),
 					value:          reflect.ValueOf(slice.ISlice(s, e, tree.state.MemoryLayout)),
-					path:           n.path,
+					path:           path.NewArrayIndex(n.subgroupOffset+s, n.path),
```

Comment:
Isn't this assigning the path to the first element in the slice? Shouldn't it be a subslice of the slice?

Also, don't the tests need updating?